py/selenium/webdriver/chromium/service.py

```diff
@@ -56,5 +58,15 @@ def __init__(
             **kwargs,
         )
 
+    @property
+    def service_args(self):
+        return self._service_args
+
+    @service_args.setter
+    def service_args(self, value):
+        if not isinstance(value, list):
```

Comment:
should probably accept a `Sequence` here instead of explicitly requiring a `list` check